### PR TITLE
UI - customer details

### PR DESCRIPTION
**Pull request by evaalves2042**

### CODE REVIEW

File: src/app/view-customer/view-customer.component.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//import { Component } from '@angular/core';
```
Comment (RafaelMeha) [inline src/app/view-customer/view-customer.component.ts:1]: Remove comment

File: src/app/customer-management/customer-management.component.spec.ts [hunk 2]
```diff
@@ -21,3 +21,4 @@ describe('CustomerManagementComponent', () => {
     expect(component).toBeTruthy();
   });
 });
+
```
Comment (Silva096) [inline src/app/customer-management/customer-management.component.spec.ts:24]: remove new line pls

File: src/app/customer-management/customer-management.component.html [hunk 3]
```diff
@@ -6,9 +6,13 @@ <h3 id="page-title">Customer Management</h3>
...
-            <td><button type="button" class="btn btn-secondary float-end btn-light">View</button></td>
+            <td><button type="button" class="btn btn-secondary float-end btn-light" [routerLink]="['view-customer', customer.id]">View</button></td>
           </tr>
         </tbody>
       </table>
     </section>
-  </div>
\ No newline at end of file
+  </div>
+  <div>
```
Comment (Silva096) [inline src/app/customer-management/customer-management.component.html:15]: remove extra div :)

File: src/app/customer-management/customer-management.component.html [hunk 4]
```diff
@@ -6,9 +6,13 @@ <h3 id="page-title">Customer Management</h3>
...
+  </div>
+
+
```
Comment (Silva096) [inline src/app/customer-management/customer-management.component.html:18]: remove new lines

File: src/app/api.service.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
```
Comment (Silva096) [inline src/app/api.service.ts:3]: Is this needed?

File: src/app/models/view-customer.ts [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export interface custumerId{
```
Comment (Silva096) [inline src/app/models/view-customer.ts:1]: custumer should be customer

File: src/app/view-customer/view-customer.component.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
+<div class="container bootstrap snippets bootdey">
+    <div class="panel-body inf-content d-flex align-items-center justify-content-center">
+        <div class="row d-flex align-items-center">
+            <div class="col-md-4">
+                <img alt="" style="width:500px;border-radius: 50%;" title="" class="img-circle img-thumbnail isTooltip" src="https://bootdey.com/img/Content/avatar/avatar7.png" data-original-title="utilizador">
```
Comment (Silva096) [inline src/app/view-customer/view-customer.component.html:5]: isToolTip is doing anything?

Comment (Silva096) [inline src/app/view-customer/view-customer.component.html:5]: Is this supposed to exist and in Portuguese? 
data-original-title="utilizador"

File: src/app/view-customer/view-customer.component.ts [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+import { ActivatedRoute } from '@angular/router';
+import { CustomerApiService } from '../services/customer-api.service';
+import { CustomerModel } from '../models/customer-management-model';
+@Component({
+  selector: 'app-view-customer',
+  templateUrl: './view-customer.component.html',
+  styleUrl: './view-customer.component.css'
+})
+
+
```
Comment (Silva096) [inline src/app/view-customer/view-customer.component.ts:11]: extra new line not needed. Leave only 1 please

File: src/app/view-customer/view-customer.component.ts [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    this.activatedRoute.params.subscribe(params => {
+      this.customerId = params['id'];
+    });
+    this.apiService.getCustomerById(this.customerId).subscribe(response => {
+      this.customerData = response
+    });
+  }
+  
+}
+
```
Comment (Silva096) [inline src/app/view-customer/view-customer.component.ts:31]: remove 7 new lines

**Review by Silva096 — CHANGES_REQUESTED** [review]
home_page_icons -> should be changed to customer_icons

**Review by Silva096 — APPROVED** [review]

**Pull request merged**